spec/features/courses/accredited_body/new_spec.rb

```diff
@@ -23,6 +23,11 @@
     stub_api_v2_build_course
     stub_api_v2_build_course(level: "primary")
 
+    stub_api_v2_request(
+      "/recruitment_cycles/2020/providers?page[page]=1",
+      resource_list_to_jsonapi([provider], meta: { count: 1 }),
+    )
+
```

Comment:
lots of test visit `signin_path` which then goes to load the providers index page